app/subscriber/src/features/my-reports/edit/settings/data-sources/ReportSectionGallery.tsx

```diff
@@ -58,20 +58,18 @@ export const ReportSectionGallery = React.forwardRef<HTMLDivElement, IReportSect
               </span>
             </Row>
           </Show>
-          <Show visible={!!section.folderId || !!section.linkedReportId}>
-            <Row>
-              <FormikCheckbox
-                name={`sections.${index}.settings.overrideExcludeHistorical`}
-                label={`Include all content from linked ${
-                  section.folderId ? 'folder' : 'report'
-                } even if in prior report`}
-              />
-              <span className="info">
-                This overrides the report option "Exclude stories that have been sent out in
-                previous report" for this section only.
-              </span>
-            </Row>
-          </Show>
+          <Row>
+            <FormikCheckbox
```

Comment:
Always show to reduce issue where the data source changes and it becomes hidden.